run-tests.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```

Comment:
minor: is this needed? why not `sh`?

Reply:
```sh``` gives Syntax error in line 19
```console
./run-tests.sh: 19: ./run-tests.sh: Syntax error: "(" unexpected
```

Reply:
Yep, it's correct - we also have [bash](https://github.com/inveniosoftware/cookiecutter-invenio-module/blob/master/%7B%7B%20cookiecutter.project_shortname%20%7D%7D/run-tests.sh#L1) here, and I vaguely remember that it was added in the last half year (i.e after this repo was generated).